{"name": "Нельзя создать заказ с невалидным хешем ингредиента", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 500\n +  where 404 = <Response [404]>.status_code", "trace": "self = <tests.test_order.TestOrder object at 0x000002E82249E390>, create_user = ({'email': 'iwmmj@yandex.ru', 'name': 'mnlaooblir', 'password': 'ffmivszpgo'}, <Response [200]>)\n\n    @allure.title('Нельзя создать заказ с невалидным хешем ингредиента')\n    @allure.description('Проверка: статус 500, Internal Server Error.')\n    def test_create_order_with_invalid_ingredient_hash(self, create_user):\n        user_data, response_post = create_user\n        token = response_post.json().get('accessToken')\n    \n        order_data = {\n            \"ingredients\": [\"invalid_hash_1\", \"invalid_hash_2\"]\n        }\n    \n        response = requests.post(f\"{url.MAIN_URL}/orders\", headers={\"Authorization\": f\"Bearer {token}\"}, json=order_data)\n    \n>       assert response.status_code == 500\nE       assert 404 == 500\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_order.py:65: AssertionError"}, "description": "Проверка: статус 500, Internal Server Error.", "start": 1724871107192, "stop": 1724871107608, "uuid": "70e2d60e-a800-4e68-b6dd-97f14961ad44", "historyId": "3b150808582f24e0cc29220ec3573b24", "testCaseId": "3b150808582f24e0cc29220ec3573b24", "fullName": "tests.test_order.TestOrder#test_create_order_with_invalid_ingredient_hash", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order"}, {"name": "subSuite", "value": "TestOrder"}, {"name": "host", "value": "LAPTOP-74V5TFPG"}, {"name": "thread", "value": "16640-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order"}]}